flink-core/src/main/java/org/apache/flink/api/java/typeutils/PojoTypeInfo.java

```diff
@@ -315,6 +331,23 @@ public int getFieldIndex(String fieldName) {
 			reflectiveFields[i] = fields[i].getField();
 		}
 
+		if (customSerializers.containsKey(this.getTypeClass())) {
+			Constructor<?>[] ctors = customSerializers.get(this.getTypeClass()).getConstructors();
+			assert ctors.length == 1;
+			try {
+				return (TypeSerializer<T>) ctors[0].newInstance(new Object[]{getTypeClass(), fieldSerializers, config});
+
+			} catch (Throwable t) {
```

Comment:
Stephan noted to me a little while back that IntelliJ convention for an unused exception is to name the variable `ignored`, so `} catch (Throwable ignored) {`.